modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,396 @@
+                "StringEquals": {
+                    "${IDP}:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager"
+                }
+            }
+        }
+    ]
+}
+EOF
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number and '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified earlier in this procedure.
++
+[NOTE]
+====
+Do not include the `https` portion of the OIDC DNS URL when replacing `{Cluster_OIDC_Endpoint}` with the OIDC DNS you identified earlier. Only the alphanumeric information that follows the `/` within the URL is needed.
+====
++
+.. Create and verify the role with the generated trusted policy:
++
+[source, terminal]
+----
+$ aws iam create-role --role-name albo-operator --assume-role-policy-document file://albo-operator-trusted-policy.json
+OPERATOR_ROLE_ARN=$(aws iam get-role --role-name albo-operator | \grep '^ROLE' | \grep -Po 'arn:aws:iam[0-9a-z/:\-_]+')
+echo $OPERATOR_ROLE_ARN
+----
++
+.. Attach the operator's permission policy to the role:
++
+[source, terminal]
+----
+$ curl -o albo-operator-permission-policy.json https://raw.githubusercontent.com/alebedev87/aws-load-balancer-operator/aws-cli-commands-for-sts/hack/operator-permission-policy.json
+aws iam put-role-policy --role-name albo-operator --policy-name perms-policy-albo-operator --policy-document file://albo-operator-permission-policy.json
+----
++
+. Generate the controller's AWS credentials:
++
+[source,terminal]
+----
+$ cat <<EOF> albo-operator-aws-credentials.cfg
+[default]
+sts_regional_endpoints = regional
+role_arn = ${OPERATOR_ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+----
++
+//[NOTE]
+//====
+//Mind the format of the credentials file, examples can be found in OCP documentation.
+//====
+
+. Create the controller's credentials secret with the generated AWS credentials:
++
+[source, terminal]
+----
+$ oc -n aws-load-balancer-operator create secret generic aws-load-balancer-operator --from-file=credentials=albo-operator-aws-credentials.cfg
+----
+
+. Create the AWS IAM policy required for the AWS Load Balancer Controller (ALBC):
++
+.. Generate a trusted policy file using your identity provider (e.g. OpenID Connect):
++
+[source,json]
+----
+IDP="<my-oidc-provider-name>"
+IDP_ARN="arn:aws:iam::<my-aws-account>:oidc-provider/${IDP}"
+cat <<EOF > albo-controller-trusted-policy.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "${IDP_ARN}"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "${IDP}:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster"
+                }
+            }
+        }
+    ]
+}
+EOF
+----
++
+.. Create and verify the role with the generated trusted policy:
```

Comment:
```suggestion
.. Create and verify the role by using the generated trusted policy:
```
- I recommend replacing `with` with `by using` because `with` could be understood to mean `placing together` in addition to `by using` -- so to my ears, `by using` is less ambiguous. (This comment applies to earlier suggestions I made, too.)